spinnaker-monitoring-third-party/third_party/prometheus_operator/setup.sh

```diff
@@ -1,16 +1,15 @@
 #!/usr/bin/env bash
 
 ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
-CONTEXT=${1:-$(kubectl config current-context)}
-
-
-echo "Adding the Spinnaker ServiceMonitor to current context: ${CONTEXT}"
-
-kubectl apply -f spinnaker-service-monitor.yaml
+CONTEXT=$(kubectl config current-context)
+NAMESPACE=${1:-default}
+KUBECTL="kubectl --context ${CONTEXT} --namespace ${NAMESPACE}"
+echo "Using context ${CONTEXT} and namespace ${NAMESPACE}."
 
+echo "Adding the Spinnaker ServiceMonitor..."
+$KUBECTL apply -f spinnaker-service-monitor.yaml
```

Comment:
Hey, you don't keep convention here.
You use `${NAMESPACE}` in brackets, but `$KUBECTL` without brackets.

Reply:
Curly brackets are used inside double quotes to prevent the variable name from becoming something unintentional. I don't technically need it in any of these cases, but it's a habit of mine. Can remove.

`$KUBECTL` is followed by a space and not inside quotes, which is why I don't do it there.